src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/soc2.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: SOC2 and New Relic
+tags:
+  - Security
+  - Security and Privacy
+  - Compliance
+metaDescription: Criteria and compliance with annual SOC 2 and FedRAMP audits by New Relic services.
+---
+
+The American Institute of Certified Public Accountants (AICPA) Service Organization Controls (SOC) reports give assurance over control environments as they relate to the retrieval, storage, processing, and transfer of data. The reports cover IT General controls and controls around availability, confidentiality and security of customer data. The SOC 2 reports cover controls around security, availability, and confidentiality of customer data. Additional information can be found at the [AICPA's Report](https://www.aicpa.org/interestareas/frc/assuranceadvisoryservices/aicpasoc2report.html).
+
+## Applicable document by service [#applicable-services]
+
+<Callout variant="caution">
+Not all [New Relic One](/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one/) services are in compliance with this program. For non-compliant services, please see the section of [services not in scope](#not-scope). 
+</Callout>
+
+The following applies to the New Relic One platform:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Document
+      </th>
+
+      <th>
+        Last Updated
+      </th>
+      
+      <th>
+        Infrastructure
+      </th>
+      
+      <th>
+        Services
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+     <tr>
+      <td>
+        SOC 2 Type 2
+      </td>
+
+      <td>
+        2021-AUG-20
+      </td>
+      
+      <td>
+        AWS & First Party
+      </td> 
+      
+      <td>
+        New Relic One Platform
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        SOC 2 Type 1
```

Comment:
@jxainguyen - same, do we have the doc?

Reply:
For all of your comments, I've connected with my director & added document links for HIPAA & ISO in a PR that needs to be merged into this. SOC document links weren't provided.